.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 dist/
 **/.DS_Store
+package-lock.json
```

Comment:
we need the package-lock.json for 2 reasons:
- we freeze versions for the connectors we use, it's important to do an explicit upgrade as the API can change. See v0.0.8 from a few days ago that change the api.
- circleCI needs the package-lock.json for caching dependencies